Getting started instructions do not work

```console
vscode@2fe9a465cc03:/workspaces/terraform-module-template/examples$ cd ../tests/
vscode@2fe9a465cc03:/workspaces/terraform-module-template/tests$  go mod init "terraform-module-template"
go: creating new go.mod: module terraform-module-template
go: to add module requirements and sums:
        go mod tidy
vscode@2fe9a465cc03:/workspaces/terraform-module-template/tests$ go test
test_hello_world.go:6:2: no required module provides package github.com/gruntwork-io/terratest/modules/terraform; to add it:
        go get github.com/gruntwork-io/terratest/modules/terraform
test_hello_world.go:7:2: no required module provides package github.com/stretchr/testify/assert; to add it:
        go get github.com/stretchr/testify/assert
```

on 

```
go version go1.18.3 linux/amd64
```

I had to run `go mod tidy` to make `go test` work.

--- a/docs/_docs/01_getting-started/quick-start.md
+++ b/docs/_docs/01_getting-started/quick-start.md
@@ -16,7 +16,7 @@ custom_js:
 
 Terratest uses the Go testing framework. To use Terratest, you need to install:
 
-- [Go](https://golang.org/) (requires version >=1.13)
+- [Go](https://golang.org/) (requires version >=1.17)
 
 ## Setting up your project
 
@@ -35,6 +35,7 @@ types of infrastructure code you can test (e.g., Packer, Kubernetes, etc).
     ```bash
     cd test
     go mod init "<MODULE_NAME>"
+    go mod tidy
     ```
 
     Where `<MODULE_NAME>` is the name of your module, typically in the format

